css/animations_transitions.css

```diff
@@ -46,6 +46,11 @@
   background: #008552;
 }
 
+.light-btn:hover {
+  transition: background-color 1s, transform 1s;
+  background: #008552;
+}
+
 /* ----- TOP MENU (Portfolio/About/Contact) ----- */
 @media screen and (min-width: 768px) {
   .list-item {
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/231652004-fb546d6a-c3f3-4d1f-bcf1-0efb6bb7cc0b.png)

- [ ] As suggested by the previous reviewer that your application has some horizontal scroll :fearful: Kindly remove that to make your project look professional and follow the Figma template :+1: